.github/ISSUE_TEMPLATE.md

```diff
@@ -1,89 +1,48 @@
-<!--
-
-Please summarize the issue in the title, and then use the template below to
-fill out the details so we can reproduce the issue on our end.
-
--->
-
-### Description
-
-[Description of the issue]
-
-### Version
 
 <!--
-
-What version of GitHub Desktop are you running? This is displayed under the
-`About GitHub Desktop` menu item. If you are running from source, include
-the commit by running `git rev-parse HEAD` from your local repository.
-
+First and foremost, we’d like to thank you for taking the time to contribute to our project. Before submitting your issue, please follow these steps:
+
+1. Familiarize yourself with our contributing guide:
+	* https://github.com/desktop/desktop/blob/master/CONTRIBUTING.md#contributing-to-github-desktop
+2. Check if your issue is in the known-issues and work-arounds doc:
+	* https://github.com/desktop/desktop/blob/master/docs/known-issues.md
+3. Make sure your issue isn’t a duplicate of another issue
```

Comment:
my biggest worry is that this is a lot of work that would probably make me doubt myself and convince myself to not post. three pre-steps (while valid) just feels like too much work. i'm happy to be convinced otherwise!

Reply:
I wanted to include this in the issue template because we didn't point out where to find info before. I think having it in the issue template makes it more likely our contribution guide will be read (or found depending on how you look at it). I also like having this info in a comment so it doesn't make the issue look messy when someone chooses to open and issue and not modify anything.